src/commands/dev.ts

```diff
@@ -418,8 +419,42 @@ export async function command(commandOptions: CommandOptions) {
     }
   });
 
-  const server = http
-    .createServer(async (req, res) => {
+
+  const readCredentials = async () => {
+    const [cert, key] = await Promise.all([fs.readFile(__dirname + '/snowpack.crt'),fs.readFile(__dirname + '/snowpack.key')])
+    
+    return {
+      cert,
+      key,
+    }
+  };
+  
+  const certify = () =>
+  execa.sync(__dirname + '/certify.sh', {
+      cwd: __dirname,
+    });
+
+    
+  let credentials
+  if (config.devOptions.secure) {
+    try {
+      credentials = await readCredentials();
+    } catch (e) {
+      certify();
```

Comment:
Generating credentials feels hardcore enough that we should either prompt the user ("credentials needed, press enter to generate") or notify the user with a slight delay ("credentials needed, generating..." + setTimeout of a second or two for the user to exit).

Reply:
So for me, coming from CRA, you just drop the `HTTPS=true` before the start command and it generates the key/cert automatically behind the scenes. It's a pretty seamless setup. I have the feeling that most people who run the app either have a cert/key they want to use or just want the app to run in HTTPS but don't care how it works.

Reply:
In that case, lets add a 1-line console log that tells the user that we're going to generate them some credentials. At the very least, it's useful to have this in the console history if something then goes wrong.